Lombiq.BaseTheme.Samples/Assets/Styles/general/_navigation.scss

```diff
@@ -0,0 +1,32 @@
+// -----------------------------------------------------------------------------
+// Navigation rules regarding the MenuWidget
+// -----------------------------------------------------------------------------
+
+@import "../abstracts/mixins";
+
+.layoutNavigation {
+    background: var(--color-background-menu);
+
+    .menuWidget {
+        &__dropdown, &__topLevel .nav-link {
+            // Another mixin from Lombiq.BaseTheme, for styling all states (e.g. visited) af an <a> link element.
+            @include on-event($self: true) {
+                color: var(--color-foreground-menu);
+            }
+        }
+
+        &__content > .navbar-nav {
+            // Sometimes you have to override rules created by some pre-made control's Bootstrap rules taht are marked as
+            // "!important" themselves so you have no choice but to use it in your own rule. Regardless, try to avoid it
+            // whenever possible.
```

Comment:
```suggestion
            // Sometimes you have to override rules created by some pre-made control's Bootstrap rules that are marked
            // as "!important" themselves so you have no choice but to use it in your own rule. Regardless, try to avoid
            // it whenever possible.
```